Shared/DatabaseDefintion/EntityModel/Repositories/ItemPictureRepository.cs

```diff
@@ -29,10 +29,16 @@ public async Task<long> AddItemPictureAsync(TM.ItemPicture picture, Cancellation
     }
 
     public async Task<IEnumerable<TM.ItemPicture>> GetItemPicturesAsync(long itemId, CancellationToken cancellationToken)
-        => await dbContext.ItemPictures.Select(ItemPictureMappings.MapItemPicture)
+    {
+        _ = await dbContext.Items.Select(ItemMappings.MapItem)
```

Comment:
Sollte der Fall nicht schon abgedeckt sein, wenn du keine Pictures für ein nicht vorhandenes Item anlegst? Also wäre es nicht schlauer den ItemId key stärker zu binden?

Weißt du was ich meine?

Reply:
Weiß nicht ganz ob ich es verstehe, aber wenn man dort ein nicht existentes Item anfragt, würde doch sonst immer eine Leere Liste zurückkommen, also um einen Fehler zu werfen, falls man ein nicht existentes Item anfragt, müssen wir das doch auf jeden Fall überprüfen. Auch wenn die ItemId stärker gebunden wäre, weil ja nicht die Datenbank das Problem ist sondern die Art unserer Filterung.

Reply:
Joa. Die Frage ist ob es ein Fehler ist für ein Item was nicht existiert ein Bild zu verlangen. Dachte dabei auch an Laufzeit. Aber da das ja eine Konstante ist und in dem Fall nicht juckt ist es mir so auch recht.